core/templates/domain/summary/exploration-summary-backend-api.service.ts

```diff
@@ -62,9 +64,8 @@ export class ExplorationSummaryBackendApiService {
       explorationIds: string[],
       includePrivateExplorations: boolean,
       successCallback: (
-        value?: ExplorationSummaryBackendDict |
-        Promise<ExplorationSummaryBackendDict>) => void,
-      errorCallback: (reason?: string | string[]) => void): void {
+        value: ExplorationSummaryBackendDict) => void,
+      errorCallback: (reason: string | null[]) => void): void {
```

Comment:
This is stacked on https://github.com/oppia/oppia/pull/13368 right ? Is the type `null[]` correct or should it be `null` ?